packages/rest/test/integration/http-handler.integration.ts

```diff
@@ -393,6 +449,115 @@ describe('HttpHandler', () => {
     }
   });
 
+  context('x-skip-body-parsing', () => {
```

Comment:
I believe `x-skip-body-parsing` was superseded by `x-parser: stream`. The tests below seem to be valid, so I think you just need to change the name of this test context?

```suggestion
  context('x-parser flag', () => {
```

Reply:
Fixed